Extension/walkthrough/installcompiler/install-compiler-windows10.md

```diff
@@ -0,0 +1,23 @@
+<h1 data-loc-id="walkthrough.windows.install.compiler">Install a C++ compiler on Windows</h1>
+<p data-loc-id="walkthrough.windows.text1">If you&#39;re doing C++ development for Windows, we recommend installing the Microsoft Visual C++ (MSVC) compiler.</p>
+<ol>
+<li><p data-loc-id="walkthrough.windows.text2">To install MSVC, open the VS Code terminal (CTRL + `) and paste in the following command:
+<pre><code style="white-space: pre-wrap;">winget install Microsoft.VisualStudio.2022.BuildTools --force --override "--wait --passive --add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows10SDK"</code></pre>
```

Comment:
The only difference between the windows10 and windows11 versions of this doc is the WindowsNNSDK component at the end of the install command.